samples/test-runner/build.gradle.kts

```diff
@@ -73,8 +80,8 @@ instrumentation {
 
                     testsCountBasedReservation {
                         device = emulator29
-                        maximum = 2
-                        minimum = 1
```

Comment:
We have some weird bug with 1 min device. Deployments generation never succeed with it

Reply:
File an issue, please